fs/nfs/rpc_clnt.c

```diff
@@ -241,6 +241,11 @@ static int rpcclnt_socket(FAR struct rpcclnt *rpc, in_port_t rport)
       goto bad;
     }
 
+  if (rpc->rc_sotype == SOCK_STREAM)
+    {
+      goto connect;
```

Comment:
According to the coding stardard, `goto` should only be used for exception handling:
https://cwiki.apache.org/confluence/display/NUTTX/Coding+Standard#goto

Reply:
Done